.watchmanconfig

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
What about this empty thing? What does this do?

Reply:
actually, i think we could delete this 🤔